<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewItem</class>
 <widget class="QDialog" name="addNewItem">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>501</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add new Materials</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>514</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_SN">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Material SN:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_MaterialSN"/>
    </item>
    <item>
     <widget class="QLabel" name="label_AssetNum">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Asset Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_AssetNum"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Name">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Material Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit__MaterialName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Brand">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Material Brand:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_MaterialBrand"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Mfr">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manufacture:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Manufacture"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Std">
      <property name="font">
       <font>
        <family>Arial Narrow</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Standard Used:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_StandardUsed"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2">
      <item>
       <widget class="QPushButton" name="pushButton_SaveAll">
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save  All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>600</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
